app/src/main/java/ru/vssemikoz/newsfeed/MainApplication.java

```diff
@@ -19,6 +26,27 @@
     private NewsApi newsApi;
     private NewsAppDataBase newsDataBase;
 
+    private Drawable whiteStarWithoutBorders;
+    private Drawable yellowStarWithoutBorders;
+    private Drawable yellowStarWithBorders;
+    private Drawable whiteStarWithBorders;
+
+    public Drawable getWhiteStarWithoutBorders() {
+        return whiteStarWithoutBorders;
+    }
+
+    public Drawable getYellowStarWithoutBorders() {
+        return yellowStarWithoutBorders;
+    }
+
+    public Drawable getYellowStarWithBorders() {
+        return yellowStarWithBorders;
+    }
+
+    public Drawable getWhiteStarWithBorders() {
+        return whiteStarWithBorders;
+    }
+
     public static Context getmContext() {
         return mContext;
```

Comment:
Please, pay attention to prefixes. If you decided to use prefix, you shouldn't include it in the get/set name.